make/checkupdates.py

```
@@ -0,0 +1,133 @@
+    else:
+        mirror = "https://archive.archlinux.org/repos/2025/11/06/$repo/os/$arch"
+
+    m = re.match(r"^(.+)\/(\d{4}\/\d{2}\/\d{2})\/\$repo\/os\/\$arch$", mirror)
+    assert m
+    current = m.group(2)
+    now = datetime.now()
+    new = f"{now.year}/{now.strftime('%m')}/{now.strftime('%d')}"
+    has_updates = False
+    if current != new:
+        url = f"{m.group(1)}/{new}/"
+        res = requests.head(url)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)